v2/apiserver/internal/system/rest/healthcheck_endpoints.go

```diff
@@ -0,0 +1,95 @@
+package rest
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"time"
+
+	"github.com/brigadecore/brigade/v2/apiserver/internal/lib/queue"
+	"github.com/brigadecore/brigade/v2/apiserver/internal/lib/restmachinery"
+	"github.com/gorilla/mux"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+// HealthcheckEndpoints implements restmachinery.Endpoints to provide
+// Healthcheck-related URL --> action mappings to a restmachinery.Server.
+type HealthcheckEndpoints struct {
```

Comment:
Naming nit. I might prefer if this were `SystemEndpoints` rather than `HealthcheckEndpoints` since it handles pings as well and those aren't really used exclusively for health-related purposes. On the flip side, it feels fine to have health check requests responded to by "system endpoints."

Again... a nit. Non-blocking. Dealer's choice on whether to act.

Reply:
Good call; renamed 👍